content/docs/05.developer-guide/10.secrets.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Managing Secrets
+---
+
+Your [flows](./1.flow.md) often need to interact with external systems. To do that, they need to programmatically authenticate using passwords or API keys.  Secrets help you securely store such variables and avoid hard-coding sensitive information within your workflow code. 
+
+You can leverage the `secret()` function, described in the [function section](./03.variables/04.function/secret.md), to retrieve the sensitive variable within your flow code.
+
+
+## Secrets in the Enterprise Edition
```

Comment:
I think you first need to document the OSS version.
Moreover, it's default not just OSS as if you don't configure a different secret bachend on the EE you will also have environment based secrets.

Reply:
agree with @loicmathieu, go at first for OSS and EE second, it's the actual life cycle of our users